datasets/dane/README.md

```diff
@@ -154,4 +157,4 @@ Data Fields:
 
```

Comment:
```suggestion

### Citation Information

```
@inproceedings{hvingelby2020dane,
  title={DaNE: A named entity resource for danish},
  author={Hvingelby, Rasmus and Pauli, Amalie Brogaard and Barrett, Maria and Rosted, Christina and Lidegaard, Lasse Malm and S{\o}gaard, Anders},
  booktitle={Proceedings of the 12th Language Resources and Evaluation Conference},
  pages={4597--4604},
  year={2020}
}
```